sklearn/preprocessing/_label.py

```diff
@@ -431,29 +440,31 @@ def fit(self, y):
         return self
 
     def fit_transform(self, y):
-        """Fit label binarizer and transform multi-class labels to binary
-        labels.
+        """
+        Fit label binarizer and transform multi-class labels to binary \
+            labels.
 
         The output of transform is sometimes referred to as
         the 1-of-K coding scheme.
 
         Parameters
         ----------
-        y : array or sparse matrix of shape [n_samples,] or \
-            [n_samples, n_classes]
-            Target values. The 2-d matrix should only contain 0 and 1,
-            represents multilabel classification. Sparse matrix can be
-            CSR, CSC, COO, DOK, or LIL.
+        y : array or sparse matrix of shape [n_samples,] or
+            [n_samples, n_classes]. Target values. The 2-d matrix
```

Comment:
Not sure it will be rendered correctly on the website.
I need to wait for the doc build to complete to check it.

Reply:
you need a break line